### PR TITLE
B3 app multimedia

### PR DESCRIPTION
**Pull request by biyeun**
Migrate `corehq.hqmedia` to bootstrap3.
will live on staging for QA.

@dimagi/product 
@orangejenny 
cc @proteusvacuum 

screen shot 2016-04-27 at 7 12 13 pm
screen shot 2016-04-27 at 7 12 21 pm
screen shot 2016-04-27 at 7 33 55 pm

### CODE REVIEW

File: corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js [hunk 1]
```diff
@@ -7,7 +9,7 @@ function MultimediaReferenceController (references, obj_map, totals) {
     self.totals = ko.observable(totals);
     
     self.toggleRefsText = ko.computed(function () {
-        return (self.showMissingReferences()) ? "Show All References" : "Show Only Missing References";
+        return (self.showMissingReferences()) ? django.gettext("Show All References") : django.gettext("Show Only Missing References");
```
Comment (dimagimon) [inline corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js]: 'django' is not defined. (no-undef)

File: corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/*globals $, _, djnago */
```
Comment (dimagimon) [inline corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js]: 'djnago' is defined but never used (no-unused-vars)

File: corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js [hunk 3]
```diff
@@ -196,15 +198,15 @@ function BaseMediaReference (ref) {
...
+        return (self.is_matched()) ? "fa fa-check media-status-ok" : "fa fa-exclamation-triangle media-status-warning";
     }, self);
 
     self.upload_button_class = ko.computed(function () {
         return (self.is_matched()) ? "btn btn-success" : "btn btn-warning";
     }, self);
 
     self.upload_button_text = ko.computed(function () {
-        return ((self.is_matched()) ? "Replace " : "Upload ") + self.media_type;
+        return ((self.is_matched()) ? django.gettext("Replace ") : django.gettext("Upload ")) + self.media_type;
```
Comment (dimagimon) [inline corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js]: 'django' is not defined. (no-undef)

File: corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js (hunk 2 above)
Comment (orangejenny) [inline corehq/apps/hqmedia/static/hqmedia/js/hqmedia.reference_controller.js]: Is there a reason to prefer either `django.gettext` or `gettext`?

File: corehq/apps/hqmedia/templates/hqmedia/references.html [hunk 4]
```diff
@@ -105,71 +121,69 @@
...
-                            </div>
-                        </div>
-                    <!-- /ko -->
+        <div class="panel-heading">
+            <h3 class="panel-title">
+                <a class="accordion-toggle"
+                     data-toggle="collapse"
+                     data-parent="#mm-reference-list"
+                     data-bind="attr: { href: '#' + id }">
+                    <i class="fa fa-angle-double-down"></i> Module: <span data-bind="text: name"></span>
```
Comment (orangejenny) [inline corehq/apps/hqmedia/templates/hqmedia/references.html]: trans

File: corehq/apps/hqmedia/templates/hqmedia/references.html [hunk 5]
```diff
@@ -105,71 +121,69 @@
...
+                </a>
+            </h3>
+        </div>
+        <div class="panel-body collapse in"
+             data-bind="attr: { id: id }">
+            <div class="accordion-inner">
+                <p>Showing all media in the module named '<span data-bind="text: name"></span>'.</p>
```
Comment (orangejenny) [inline corehq/apps/hqmedia/templates/hqmedia/references.html]: trans

File: corehq/apps/hqmedia/templates/hqmedia/references.html [hunk 6]
```diff
@@ -105,71 +121,69 @@
...
+                <div class="accordion" data-bind="attr: { id: id + '-ModuleRefs' }">
+                    {% include "hqmedia/partials/reference_table.html" with ref_type="active_menu_references" show_refs="showMenuRefs" parent_accordion="ModuleRefs" this_accordion="ModuleMenuRefs" title="Menu References" %}
                 </div>
+                <!-- ko foreach: forms -->
+                    <div data-bind="visible: showForm">
+                        <h4>Form: <span data-bind="text: name"></span></h4>
```
Comment (orangejenny) [inline corehq/apps/hqmedia/templates/hqmedia/references.html]: trans

File: corehq/apps/hqmedia/templates/hqmedia/uploader/queue_multi.html [hunk 7]
```diff
@@ -3,8 +3,8 @@
     <td><%= file_name %></td>
     <td><%= file_size %> MB</td>
     <td class="hqm-progress">
-        <div class="progress progress-striped active" style="margin-bottom: 5px;">
-            <div class="bar" style="width: 0%;"></div>
+        <div class="progress" style="margin-bottom: 5px;">
+            <div class="bar progress-bar progress-bar-striped active" style="width: 0%;"></div>
```
Comment (orangejenny) [inline corehq/apps/hqmedia/templates/hqmedia/uploader/queue_multi.html]: can drop `.bar`, I think

File: corehq/apps/hqmedia/templates/hqmedia/uploader/queue_single.html [hunk 8]
```diff
@@ -1,24 +1,23 @@
...
     <div class="alert alert-success" style="margin-bottom: 2px; margin-top: 10px;">
-        <i class="icon-ok fa fa-check"></i> <%= file_name %> <strong>[<%= file_size %> MB]</strong>
+        <i class="fa fa-check"></i> <%= file_name %> <strong>[<%= file_size %> MB]</strong>
     </div>
     <div class="hqm-progress">
-        <div class="progress progress-striped active" style="margin-bottom: 5px;">
-            <!-- .bar include for Bootstrap 2 compatibility -->
-            <div class="bar progress-bar" style="width: 0%;"></div>
+        <div class="progress" style="margin-bottom: 5px;">
+            <div class="bar progress-bar progress-bar-striped active" style="width: 0%;"></div>
```
Comment (orangejenny) [inline corehq/apps/hqmedia/templates/hqmedia/uploader/queue_single.html]: drop `.bar`

**Comment by orangejenny** [comment]
Looks good, just some minor comments. Holding off on merge until QA happens.

**Comment by biyeun** [comment]
@orangejenny QA passed cc @nickpell

**Pull request merged**